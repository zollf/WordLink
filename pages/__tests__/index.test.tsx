import React from 'react';
import useWithStores from '../../test/useWithStores';

import { Index } from '..';

import renderer from 'react-test-renderer';

const component = useWithStores(<Index />);

describe('<Index />', () => {
  it('matches snapshot', () => {
    expect(renderer.create(component)).toMatchSnapshot();
  });
});
