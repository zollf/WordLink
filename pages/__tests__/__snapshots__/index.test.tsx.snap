// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> matches snapshot 1`] = `
<View
  style={
    Object {
      "height": "100%",
    }
  }
>
  <View
    data-test-id="wrapper"
    style={
      Object {
        "alignItems": "center",
        "bottom": 106.5,
        "display": "flex",
        "justifyContent": "center",
        "position": "absolute",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoLinearGradient
      proxiedProperties={
        Object {
          "colors": Array [
            4294177779,
            4293125091,
          ],
          "data-test-id": "menu-bar",
          "endPoint": undefined,
          "locations": Array [
            0.5,
            0.5,
          ],
          "startPoint": Array [
            -0.1,
            0.1,
          ],
        }
      }
      style={
        Object {
          "alignItems": "center",
          "borderRadius": 5,
          "display": "flex",
          "flexDirection": "row",
          "height": 75,
          "justifyContent": "center",
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 2,
            "width": 2,
          },
          "shadowOpacity": 0.1,
          "shadowRadius": 4,
          "width": 212,
          "zIndex": -1,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderRadius": 5,
            "display": "flex",
            "height": 50,
            "justifyContent": "center",
            "marginLeft": 10,
            "marginRight": 10,
            "opacity": 0.5,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 2,
              "width": 2,
            },
            "shadowOpacity": 0.1,
            "shadowRadius": 4,
            "width": 50,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 40,
              "width": 40,
            }
          }
        />
      </View>
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderRadius": 5,
            "display": "flex",
            "height": 50,
            "justifyContent": "center",
            "marginLeft": 10,
            "marginRight": 10,
            "opacity": 0.5,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 2,
              "width": 2,
            },
            "shadowOpacity": 0.1,
            "shadowRadius": 4,
            "width": 50,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 40,
              "width": 40,
            }
          }
        />
      </View>
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderRadius": 5,
            "display": "flex",
            "height": 50,
            "justifyContent": "center",
            "marginLeft": 10,
            "marginRight": 10,
            "opacity": 0.5,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 2,
              "width": 2,
            },
            "shadowOpacity": 0.1,
            "shadowRadius": 4,
            "width": 50,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 40,
              "width": 40,
            }
          }
        />
      </View>
      <View
        style={
          Array [
            Object {
              "borderRadius": 5,
              "height": 50,
              "position": "absolute",
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 2,
              },
              "shadowOpacity": 0.1,
              "shadowRadius": 4,
              "width": 50,
              "zIndex": -1,
            },
            Object {
              "left": undefined,
            },
          ]
        }
      >
        <ViewManagerAdapter_ExpoLinearGradient
          proxiedProperties={
            Object {
              "colors": Array [
                4282711882,
                4280928549,
              ],
              "endPoint": undefined,
              "locations": undefined,
              "startPoint": undefined,
            }
          }
          style={
            Object {
              "borderRadius": 5,
              "height": 50,
              "position": "absolute",
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 2,
              },
              "shadowOpacity": 0.1,
              "shadowRadius": 4,
              "width": 50,
              "zIndex": -1,
            }
          }
        />
      </View>
    </ViewManagerAdapter_ExpoLinearGradient>
  </View>
</View>
`;
